### DISCUSSION TITLE
Display after x number of page views.

### DISCUSSION THREAD

**Discussion by du8die** [formbricks/formbricks#4403] [General Requests]
On an embedded survey, is there a way to only show the survey to a user after they have visited, say 10 pages?  I'm trying to use the "did you accomplish your goal" survey - and it's popping up right when a user hits the first page of the site.

Thanks!~

**Comment by Dhruwang** [comment]
Hey @du8die , by embedded survey, do you mean the embedded link surveys or app/website survey ?

**Reply by du8die** [reply]
App/Website. 

Thanks!

**Reply by Dhruwang** [reply]
You can use code actions in this scenario, which are available in the survey trigger settings. With code actions, you can trigger a survey directly from your code. To achieve this, you would need to implement a page visit counter in your app to track when the survey should be displayed. Let me know if it helps or you need another solution 😊

<img width="720" alt="Screenshot 2024-12-10 at 12 04 05 PM" src="https://github.com/user-attachments/assets/4fd37170-acd3-4d30-a906-426b58616489">
